src/main/java/org/folio/service/dataimport/handlers/CreateOrderEventHandler.java

```diff
@@ -159,6 +167,24 @@ private Map<String, String> extractOkapiHeaders(DataImportEventPayload dataImpor
     return headers;
   }
 
+  private Future<String> generateSequentialOrderId(DataImportEventPayload dataImportEventPayload, JsonObject tenantConfig) {
+    LOGGER.debug("generateSequentialOrderId :: jobExecutionId: {} ", dataImportEventPayload.getJobExecutionId());
+
+    String tenantId = dataImportEventPayload.getTenant();
+    if (isEmpty(tenantConfig.getString(PO_LINES_LIMIT_PROPERTY))) {
+      return Future.failedFuture(new IllegalArgumentException("Parameter poLines-limit is missing."));
```

Comment:
@Aliaksandr-Fedasiuk Let's add in the log at least jobExecutionId

Reply:
propose not to fail event processing and instead to use 1 as the default poLines limit, if the parameter is absent (like logic in the mod-orders services)

Reply:
done